source/_integrations/tuya.markdown

```diff
@@ -25,79 +24,119 @@ ha_platforms:
 ha_dhcp: true
 ---
 
-The `tuya` integration is the main integration to integrate [Tuya Smart](https://www.tuya.com) related platforms, except the Zigbee and Bluetooth(BLE) hub. This includes devices linked with the Tuya, Smart Life, and Jinvoo Smart apps. You will need your account information (username, password and account country code) from one of these apps to discover and control devices which related to your account.
+The Tuya Home Assistant integration is developed for controlling **Powered by Tuya (PBT)** devices using the [tuya-iot-python-sdk](https://github.com/tuya/tuya-iot-python-sdk) (a Python version of [Tuya Open API](https://developer.tuya.com/en/docs/iot/api-reference?id=Ka7qb7vhber64)), which is officially maintained by the Tuya Developer Team.
```

Comment:
Current sentence only mentions the SDK is official. I would leave out the SDK and emphasize that this is an official one.

```suggestion
The Tuya Home Assistant integration allows you to control any **Powered by Tuya** devices. This integration is official and maintained by Tuya.
```

Reply:
fixed